src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/ExcelAdaptorServiceImpl.java

```diff
@@ -105,22 +115,36 @@ private void validateHeaders(List<String> headersToBeValidated,
         Field rowField = getRowIdField((Class<Object>) classType);
         while (rowIterator.hasNext()) {
             Row row = rowIterator.next();
-            //Skipping empty rows
-            if (checkIfRowIsEmpty(row)) {
-                continue;
+            Object bean = null;
+            try {
+                if (checkIfRowIsEmpty(row)) {
+                    continue; //Skipping empty rows
+                }
+                bean = populateDomainObject(classType, rowField, headers, row, parentFieldMap, childHeaderToCellMap,
```

Comment:
can we please refactor it in method will be more readable

Reply:
refactored